docs/content/v0.20/contributor.md

```diff
@@ -58,3 +58,21 @@ Checking KubeStellar pre-requisites:
 X ArgoCD CLI
   how to install: https://argo-cd.readthedocs.io/en/stable/cli_installation/
 ```
+
+## Integration testing
+
+There is currently one integration test. Contributors can run it. It is not (yet) run in CI.
+
+This test requires you to already have `etcd` on your `$PATH`.
+See https://github.com/kubernetes/kubernetes/blob/v1.28.2/hack/install-etcd.sh for an example of how to do that.
+
+To run the test, issue a command like the following.
+
+```shell
+CONTROLLER_TEST_NUM_OBJECTS=24 go test -v ./test/integration/controller-manager &> /tmp/test.log

```

Comment:
don't we want to add a Make target and just have `make integration-test`  ?

Reply:
My goal here is to get the contributor directly to the running of this test. No more machinery is required, so I did not choose to add any requirement to use more machinery here. See #1784 for a PR that adds the machinery necessary to run this test in CI. That is most naturally a script. As a Makefile target it would be a PHONY, not really leveraging `make`.